app/controllers/group_memberships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GroupMembershipsController < AuthenticationController
+  def create
+    result = CreateGroupMembership.call(user_id:  current_user.id,
+                                        group_id: params[:group_id])
+    if result.success?
+      head :no_content
```

Comment:
@Enriikke `:no_content` seems the most logical? I think the front-end will just respond to a successful create with a redirect and a success message...